starter-code/Bookshelf/app/src/main/java/ly/generalassemb/drewmahrt/bookshelf/MainActivity.java

```diff
@@ -29,7 +35,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         //Use helper method to add books to the list
-        mBookList = generateBooks();
+        mBookList = (ArrayList)generateBooks();
```

Comment:
If you declare `mBookList` as type `List` instead of `ArrayList` then you won't have to cast the return value from `generateBooks()` here